cosmos/dbt/parser/project.py

```diff
@@ -206,6 +206,15 @@ def __post_init__(self) -> None:
         """
         Initializes the parser.
         """
+        if self.dbt_root_path is None:
```

Comment:
Why did we change to set the default value in the constructor?

Reply:
Discussed in meeting.